float32 integration_time
---
bool success
